README.md

```diff
@@ -2,24 +2,17 @@
 To learn more about using Terraform with PagerDuty, checkout [The How and Why of Using Terraform with PagerDuty](https://www.pagerduty.com/eng/how-why-terraform/).
 
 ## How to Use
-- Sign up for a [Developer Account](https://developer.pagerduty.com/sign-up/). This account will provide you with a sandbox that will allow you to build and modify things in a PagerDuty account without messing with production data.
+- Sign up for a [Account](https://www.pagerduty.com/sign-up/). This account will provide you with a 14 days trial that will allow you to build and modify things in a PagerDuty account without messing with production data. It will also allow you to have unlimited users and play with some advanced functionality that a developer account won't let you use. 
```

Comment:
Should we rename the repo if we make this change?

Or should we document regular and dev accounts as alternative options for different use cases?

Reply:
I think the makes sense (renaming), but we aren't doing it today.

Long term goal is to make a dev account friendly version that people can use but we didn't' get it done.